### PR TITLE
Fix balances subtab inside Assets tab

### PR DESCRIPTION
**Pull request by KarishmaBothara**
Fixes portal assets/balances when using with DevNet

### CODE REVIEW

File: packages/page-assets/src/Balances/Account.tsx [hunk 1]
```diff
@@ -26,17 +26,14 @@ interface Props {
   siFormat: [number, string];
 }
 
-function Account ({ account: { balance, isFrozen, reason, sufficient }, accountId, assetId, className, minBalance, siFormat }: Props): React.ReactElement<Props> {
+function Account ({ account: { balance, reason, sufficient }, accountId, assetId, className, minBalance, siFormat }: Props): React.ReactElement<Props> {
```
Comment (zees-dev) [inline packages/page-assets/src/Balances/Account.tsx:29]: typo?
```suggestion
function Account ({ account: { balance, reason, sufficient }, accountId, assetId, className, minBalance, isFormat }: Props): React.ReactElement<Props> {
```

File: packages/page-assets/src/Balances/useBalances.ts [hunk 2]
```diff
@@ -79,6 +85,10 @@ function useBalancesImpl (id?: BN | null): AccountResult[] | null {
   const balancesQuery = useCall(isBalances && api.query.system.account.multi, [allAccounts], BALANCES_OPTS);
   const assetsQuery = useCall(keys && api.query.assets.account.multi, keys, ASSETS_OPTS);
 
+  const runtimeVersion = useCall<RuntimeVersion>(api.rpc.state.subscribeRuntimeVersion);
+
+  specVersion = runtimeVersion?.specVersion?.toNumber();
```
Comment (zees-dev) [inline packages/page-assets/src/Balances/useBalances.ts:90]: not clear how these vars are used in the function

**Review by ken-futureverse — APPROVED** [review]
LGTM

**Review by zees-dev — APPROVED** [review]
LGTM 👍 

Minor comments; shouldn't be blocking though.

File: packages/page-assets/src/Balances/Account.tsx (hunk 1 above)
Comment (KarishmaBothara) [reply packages/page-assets/src/Balances/Account.tsx:29]: no.. this is siFormat (formatting numbers according to SI standards)

**Pull request merged**